pkgs/os-specific/darwin/darwin-stubs/default.nix

```diff
@@ -1,20 +1,18 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchurl }:
 
 stdenv.mkDerivation {
   pname = "darwin-stubs";
   version = "10.12";
 
-  src = fetchFromGitHub {
-    owner = "NixOS";
-    repo = "darwin-stubs";
-    rev = "80b3d4a57d3454c975eefd984c804dbd76f04ef2";
-    sha256 = "0sslg4rmskms8ixixv1gvnrvvvmn723vbfjj6mcn24fj2ncg38y7";
+  src = fetchurl {
+    url = "https://s3.ap-northeast-1.amazonaws.com/nix-misc.cons.org.nz/stubs-10.12-4ea7110d947803a4a2dec334a630587fe049a7b5.tar.gz";
```

Comment:
Can we use the github releases here?

Reply:
Ah, sorry, yes, that is the plan, I added this for testing. Related: is there a preferred versioning scheme for tags for the repo? I'll make one and update this.

Reply:
Not really. Something tied to the release versions would be nice but we can't make a direct mapping since other things can change so that doesn't seem like a great idea. Just semver or something date based probably makes the most sense.

Reply:
I've replaced this with a github archive url.